masschallenge/impact-api
# AC-4836 - Expand support for django-accelerator objects in impact-api

to test:
- checkout this branch
- load up the initial schema from the latest accelerate app via ~```make loaddb```~ **Y.M. -  Updated command is now `make load-db` or `make load-remote-db`.**
- visit the django-accelerate app's root view at localhost:8000/api/accelerator/
- verify that everything is functional and that objects can be created, updated, deleted just as with the impact objects. Also verify that OPTIONS calls return a schema for a given object.
